v2/components/schemas/Warning.yaml

```
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-3.0
+
+# This schema describes a single warning object.
+type: object
+properties:
+  message:
+    type: string
+    description: The warnings message content.
+  first-seen:
+    type: string
+    format: date-time
+    description: The first time a message of this type was created (RFC3339 UTC format).
+  last-seen:
```

```suggestion
  last-added:
```

See: https://github.com/canonical/snapd/blob/master/overlord/state/warning.go#L43-L50